<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DD5B-D-1CFJWQ1" name="r09_SignupandLogin_Chrome.Trident_Profile_change_UserName_Duplicate_Email_98974" tests="1" failures="1" timestamp="5 Oct 2016 05:21:50 GMT" time="23.096" errors="0">
  <testcase name="Profile_change_UserName_Duplicate_Email" time="23.096" classname="r09_SignupandLogin_Chrome.Trident_Profile_change_UserName_Duplicate_Email_98974">
    <failure type="java.lang.AssertionError" message="expected [no] but found [yes]">
      <![CDATA[java.lang.AssertionError: expected [no] but found [yes]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:178)
	at org.testng.Assert.assertEquals(Assert.java:188)
	at r09_SignupandLogin_Chrome.Trident_Profile_change_UserName_Duplicate_Email_98974.Profile_change_UserName_Duplicate_Email(Trident_Profile_change_UserName_Duplicate_Email_98974.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at integration.testrail.Integration.main(Integration.java:33)
]]>
    </failure>
  </testcase> <!-- Profile_change_UserName_Duplicate_Email -->
</testsuite> <!-- r09_SignupandLogin_Chrome.Trident_Profile_change_UserName_Duplicate_Email_98974 -->
